### PR TITLE
Storage tests

### PR DESCRIPTION
**Pull request by griff-pyrofex**

### CODE REVIEW

**Comment by griff-pyrofex** [comment]
This sumbission includes the code that tests classes Lmdb and Storage, including query unification.

File: storage/src/test/scala/coop/rchain/storage/StorageTests.scala [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    println("putGetDupSort() end")
+  }
+
+
+  def loadFiles():Unit = {
+
+    println("loadFile() begin")
+
+    val storeFlat =
+      "/home/griff/Documents/storage/2017-10-19_2_lmdbjava/stores/storeFlat.txt"
```
Comment (kirkwood) [inline storage/src/test/scala/coop/rchain/storage/StorageTests.scala]: LIES! :)

Comment (griff-pyrofex) [reply storage/src/test/scala/coop/rchain/storage/StorageTests.scala]: What do you think of:

    val storeFlat =
      "<root>/src/test/scala/coop/rchain/storage/stores/storeFlat.txt"
    val storeNested =
      "<root>/src/test/scala/coop/rchain/storage/stores/storeNested.txt"
    val storeRecursive =
      "<root>/src/test/scala/coop/rchain/storage/stores/storeRecursive.txt"

I guess I could write the code that figures out what <root> is.

Comment (griff-pyrofex) [reply storage/src/test/scala/coop/rchain/storage/StorageTests.scala]: My last sentence got rendered in Markdown in an unexpected way.  Here is my second attempt:

I guess I could write the code that figures out what <<root>> is.

Comment (griff-pyrofex) [reply storage/src/test/scala/coop/rchain/storage/StorageTests.scala]: Still didn't work.  Third attempt:

I guess I could write the code that figures out what "root", with less-than-symbol on the left and greater-than-symbol on the right, is.

Comment (kirkwood) [reply storage/src/test/scala/coop/rchain/storage/StorageTests.scala]: Heh. I think I see what you're going for.

Is this runnable via `sbt test`, yet? If so, then I'd put in relative paths (that can be `realpath`'d if necessary) that let it work from wherever `sbt test` runs.

**Comment by griff-pyrofex** [comment]
The code runs with sbt in a single directory but not in a directory
structure.  I expect to have time to work on that today.

**Pull request merged**